docs/source/user_guide/faq.rst

```diff
@@ -0,0 +1,118 @@
+fastplotlib FAQ
+===============
+
+What is `fastplotlib`?
+----------------------
+
+    `fastplotlib` is a scientific plotting library built on top of the `pygfx <https://github.com/pygfx/pygfx>`_ rendering engine
+    that leverages new graphics APIs and modern GPU hardware to create fast and interactive visualizations.
+
+
+What can I do with `fastplotlib`?
+---------------------------------
+
+    `fastplotlib` allows for:
+        - GPU accelerated visualization
+        - interactive visualization via an intuitive and expressive API
+        - rapid prototyping and algorithm design
+        - easy exploration and fast rendering of large-scale data
+        - design, develop, evaluate and ship machine learning models
+
+Do I need a GPU?
+----------------
+
+    Technically no, you can perform limited software rendering on linux using lavapipe.
+    However, `fastplotlib` is intentionally built for realtime rendering using the latest GPU technologies, so we
+    strongly recommend that you use a GPU. With that being said, an integrated GPU should be sufficient for many
+    use-cases.
+
+How does `fastplotlib` relate to `matplotlib`?
+----------------------------------------------
+
+    `fastplotlib` is **not** related to `matplotlib` in any way.
+
+    These are two completely different libraries with their own APIs and use-cases. The `fastplotlib` library is primarily for *interactive*
+    visualization that runs on the GPU using WGPU. The `fastplotlib` architecture is completely different from `matplotlib`. Using `fastplotlib`
+    is more akin to using `numpy`. See the "How can I learn to use `fastplotlib`?" section below.
```

Comment:
expand on this a bit more, ideas from: https://github.com/fastplotlib/fastplotlib/issues/122#issuecomment-1399890462 

The pygfx buffer interface is really unlike anything in matplotlib and other libs which is a major reason for why fpl can have an array-like API for plotting!